index.html

```diff
@@ -135,5 +133,32 @@ <h3 class="ab-title">Languages</h3>
     </div>
 
   </section>
+  <footer>
+    <div id="contact-us" class="col-center container-3 column">
+      <h2 class="tema-2-2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <form action="https://formspree.io/f/xaykndkq 
+      " method="post">
+        <div class="column col-center">
+          <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" class="fill-h spaced-top input-1" required>
+          <input type="email" name="email" id="email" placeholder="Email address" class="fill-h spaced-top input-1" required>
+          <textarea name="message" id="message" placeholder="Enter text here" cols="30" rows="10" maxlength="500" class="fill-h spaced-top t-area-1" required></textarea>
+          <button type="submit" class="link-btn spaced-top">Get in touch</button>
+        </div>
+      </form>
+    </div>
+    <div class=" container-4 column col-center">
+      <div class="divider-4"></div>
+      <div class="col-center">
+        <ul>
+          <li><a href="https://github.com/"><img class="icon" src="icons/Icon-GitHub2.png" alt="Icon-GitHub"></a></li>
+          <li><a href="https://www.linkedin.com/"><img class="icon" src="icons/Icon-Linkedin2.png" alt="Icon-Linkedin"></a></li>
+          <li><a href="https://angel.co/"><img class="icon" src="icons/Icon-Angelist2.png" alt="Icon-Angelist"></a></li>
+          <li><a href="https://twitter.com/"><img class="icon" src="icons/Icon-Twitter2.png" alt="Icon-Twitter"></a></li>
+          <li><a href="https://medium.com/"><img class="icon" src="icons/Icon-Medium2.png" alt="Icon-Medium"></a></li>
```

Comment:
- Kindly make sure you use the correct path to display icon images. as you check in the picture below your icon are not visible, fix that, please or you can even import the library for icons and use icons instead of images :grinning:  :grinning:  :grinning: 
## your design 
![image](https://user-images.githubusercontent.com/95297251/186993464-bd6f3f40-078f-4152-8962-a22d9f15a7e4.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/186994033-b043bc9f-d873-451d-ad84-25e3b26a2079.png)